app.py

```diff
@@ -0,0 +1,17 @@
+from flask import Flask, jsonify
+from chatbot import message_to_bot, setup
+
+app = Flask(__name__)
+
+@app.route('/chatbot/<user_input>', methods=['GET'])
+def chat(user_input):
+    clf, learn_response = setup()
```

Comment:
Ideally, any configuration should be done outside any of the functions linked to paths. AFAIK, keeping it here would initiate a setup every time this path is hit since there is no implicit caching.

Reply:
Good point! Thanks for pointing this out :)